### PR TITLE
Added code for radio map estimation with autonomous UAVs.

### PR DESCRIPTION
**Pull request by raju-stha**

### CODE REVIEW

**Comment by fachu000** [comment]
Thanks Raju. There seems to be no README file or any other file explaining how to run the simulations, right? Then, the user will not be able to run this...

**Comment by raju-stha** [comment]
Yes, it is important to have READme file. Okay, I will provide basic guidelines for execution of the files.

**Comment by fachu000** [comment]
> Yes, it is important to have READme file. Okay, I will provide basic guidelines for execution of the files.

Sounds good. Thanks.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+The experiments reproducing different figures in the paper are organized in the methods located in the file `Experiments/route_planning_experiments.py`.
+The comments before each method indicate which figure(s) on the paper it generates.
+For example, to run the experiment no 1002 in the `Experiments/route_planning_experiments.py`, in the command prompt, execute the command `$ python run_experiment.py 1002`.
+
+For more information about the simulation environment, please check [here](https://github.com/fachu000/GSim-Python).
+
+
+# Citation
+If our code is helpful in you research or work, please cite our paper:
+........
```
Comment (fachu000) [inline README.md]: This seems incomplete. I suggest you also write that they may send you an email if they have trouble running the code.

**Review by fachu000 — CHANGES_REQUESTED** [review]
Nice job! just a small change and I will merge.

**Pull request merged**